packages/block-serialization-default-parser/class-wp-block-parser-frame.php

```diff
@@ -65,8 +65,8 @@ class WP_Block_Parser_Frame {
 	 * @param WP_Block_Parser_Block $block              Full or partial block.
 	 * @param int                   $token_start        Byte offset into document for start of parse token.
 	 * @param int                   $token_length       Byte length of entire parse token string.
-	 * @param int                   $prev_offset        Byte offset into document for after parse token ends.
-	 * @param int                   $leading_html_start Byte offset into document where leading HTML before token starts.
+	 * @param int|null              $prev_offset        Byte offset into document for after parse token ends. Default null.
+	 * @param int|null              $leading_html_start Byte offset into document where leading HTML before token starts. Default null.
```

Comment:
If we're here making this change it might be helpful to also include an `Optional. ` prefix, as this pattern is predominant in other code.

Could you add that to the start of each of the parameter descriptions?

Reply:
@dmsnell Thank you for the review, I've added the `Optional.` prefix descriptions in the latest commit.